packages/mdc-theme/README.md

```diff
@@ -16,27 +16,33 @@ path: /catalog/theme/
   </a>
 </div>-->
 
-MDC Theme is a foundational module that provides theming to MDC-Web components, and also makes it available to
-developers as Sass functions and mixins, as CSS custom properties, and as a set of CSS classes.
+This color palette comprises primary and accent colors that can be used for illustration or to develop your brand colors.
```

Comment:
super nit: feel if this sentence is repeating itself with the next one. Feel free to neglect if you disagree.